infra/modules/database/role-manager.tf

```diff
@@ -79,6 +84,11 @@ resource "aws_kms_key" "role_manager" {
   enable_key_rotation = true
 }
 
+data "aws_secretsmanager_secret" "db_pass" {
+  count = length(aws_rds_cluster.db.master_user_secret)
```

Comment:
@charles-nava @shawnvanderjagt I don't think we want the count here. See https://github.com/navapbc/template-infra/issues/475 as discovered by @rocketnova